build/configs/rtl8730e/scripts/rlx8730e_img2_ddr.ld

```diff
@@ -0,0 +1,357 @@
+
+	/* KM4 Non-secure RAM for XIP only */
+	KM4_BD_RAM (rwx) :                      ORIGIN = 0x20014020, LENGTH = 0x2001B000 - 0x20014020	/* KM4 MAIN SRAM: 28k */
+
+	/* KM4 EXTENTION SRAM */
+	EXTENTION_SRAM (rwx) :                  ORIGIN = 0x22000000, LENGTH = 0x23000000 - 0x22000000	/* KM4 EXTENTION SRAM: 16384k (Wi-Fi 40KB + Bluetooth 256KB) */
+
+	/* Retention RAM */
+	RETENTION_RAM (rwx) :                   ORIGIN = 0x23020000, LENGTH = 0x23020200 - 0x23020000	/* KM0 Retention SRAM: 512B */
+
+	/* KM0 ROM */
+	KM0_IROM (rx) :                         ORIGIN = 0x00000000, LENGTH = 0x0000C000 - 0x00000000	/* KM0 IROM: 48k */
+	KM0_DROM (rx) :                         ORIGIN = 0x0000C000, LENGTH = 0x00010000 - 0x0000C000	/* KM0 DROM: 16k */
+
+	/* KM0 RAM */
+	KM0_ROMBSS_RAM (rw) :                   ORIGIN = 0x23000000, LENGTH = 0x23001000 - 0x23000000	/* KM0 ROM BSS RAM: 4k */
+	KM0_MSP_RAM (rw) :                      ORIGIN = 0x23001000, LENGTH = 0x23002000 - 0x23001000	/* KM0 MSP RAM: 4k */
+	KM0_BD_RAM (rwx) :                      ORIGIN = 0x23002020, LENGTH = 0x2301FD00 - 0x23002020	/* KM0 MAIN RAM: 119k */
+	KM0_IPC_RAM (rwx) :                     ORIGIN = 0x2301FD00, LENGTH = 0x23020000 - 0x2301FD00	/* KM0 IPC RAM: 768B */
+
+	/* Flash */
+	KM4_BOOT_XIP (rx) :                     ORIGIN = 0x0A000020, LENGTH = 0x0C000000 - 0x0A000020	/* KM4 Bootloader Virtual address: 32767k */
+	KM0_IMG2_XIP (rx) :                     ORIGIN = 0x0C000020, LENGTH = 0x0E000000 - 0x0C000020	/* KM0 IMG2 Virtual address: 32767k */
+	KM4_IMG2_XIP (rx) :                     ORIGIN = 0x0D000020, LENGTH = 0x0E000000 - 0x0D000020	/* KM4 IMG2 Virtual address: 32767k */
+	CA32_IMG2_XIP (rx) :                    ORIGIN = 0x0E000020, LENGTH = 0x10000000 - 0x0E000020	/* KM0 IMG2 Virtual address: 32767k */
+
+	BTRACE (rx) :                           ORIGIN = 0xC8800000, LENGTH = 0xC9000000 - 0xC8800000	/* BT LOG Virtual address: 8192k */
+
+	/* CA32 ROM */
+	CA32_IROM (rx) :                        ORIGIN = 0x00000000, LENGTH = 0x00000400 - 0x00000000	/* KM0 IROM: 1k */
+
+	/* CA32 RAM */
+	CA32_BL1_SRAM_S (rwx) :                 ORIGIN = 0x3001FE00, LENGTH = 0x30020000 - 0x3001FE00	/* CA32 BL1 SRAM: 512B */
+	CA32_BL1_DRAM_S (rwx) :                 ORIGIN = 0x70017020, LENGTH = 0x70037000 - 0x70017020	/* CA32 BL1 DRAM S: 127k */
+	CA32_FIP_DRAM_NS (rwx) :                ORIGIN = 0x60200000, LENGTH = 0x60400000 - 0x60200000	/* CA32 FIP(BL2+BL32+BL33) IMG: 2MB */
+	CA32_BL3_DRAM_NS (rwx) :                ORIGIN = 0x60100000, LENGTH = 0x64000000 - 0x60100000	/* CA32 BL3 DRAM NS: 63MB DDR2 */
+	KM0_BD_DRAM (rwx) :                     ORIGIN = 0x6FFFFFFF, LENGTH = 0x6FFFFFFF - 0x6FFFFFFF	/* KM0 BD DRAM */
+	KM4_DRAM_HEAP_EXT (rwx) :               ORIGIN = 0x6FFFFFFF, LENGTH = 0x6FFFFFFF - 0x6FFFFFFF	/* KM4 PSRAM HEAP EXT */
+
+	CA32_FLASH_EXT (r) :                    ORIGIN = 0x09000000, LENGTH = 0x09020000 - 0x09000000	/* 128KB of external flash */

```

Comment:
Could you please set external flash address 0x0A000000 ~ 0x0E000000 (64MB)

Reply:
This will be removed, rtk data binary will be merged into resource binary. We will not need this anymore.
You may check https://github.com/Samsung/TizenRT/pull/6303